delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -0,0 +1,289 @@
+import ch.epfl.bluebrain.nexus.delta.rdf.utils.JsonKeyOrdering
+import ch.epfl.bluebrain.nexus.delta.routes.AclsRoutes.PatchAcl._
+import ch.epfl.bluebrain.nexus.delta.routes.AclsRoutes._
+import ch.epfl.bluebrain.nexus.delta.routes.marshalling.{CirceUnmarshalling, QueryParamsUnmarshalling}
+import ch.epfl.bluebrain.nexus.delta.sdk.directives.AuthDirectives
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceF._
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclAddress.{Organization, Project}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.AclAddressFilter.{AnyOrganization, AnyOrganizationAnyProject, AnyProject}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.acls.{Acl, AclAddress, AclAddressFilter, AclRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.Permission
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchResults.{encodeResults, searchResultsJsonLdEncoder, SearchEncoder}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{BaseUri, Label, ResourceF}
+import ch.epfl.bluebrain.nexus.delta.sdk.syntax._
+import ch.epfl.bluebrain.nexus.delta.sdk.{AclResource, Acls, Identities, Lens}
+import io.circe._
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.syntax.EncoderOps
+import kamon.instrumentation.akka.http.TracingDirectives.operationName
+import monix.execution.Scheduler
+
+class AclsRoutes(identities: Identities, acls: Acls)(implicit
+    baseUri: BaseUri,
+    s: Scheduler,
+    cr: RemoteContextResolution,
+    ordering: JsonKeyOrdering
+) extends AuthDirectives(identities, acls)
+    with DeltaDirectives
+    with CirceUnmarshalling
+    with QueryParamsUnmarshalling {
+
+  import baseUri._
+
+  private val aclsIri = endpoint.toIri / "acls"
+
+  private val any = "*"
+
+  private val simultaneousRevAndAncestorsRejection =
+    MalformedQueryParamRejection("rev", "rev and ancestors query parameters cannot be present simultaneously")
+
+  implicit val iriLens: Lens[AclAddress, Iri] = {
+    case AclAddress.Root    => aclsIri
+    case Organization(org)  => aclsIri / org.value
+    case Project(org, proj) => aclsIri / org.value / proj.value
+  }
+
+  implicit val aclContext: ContextValue = Acl.context
+
+  implicit val aclResponseEncoder: Encoder.AsObject[AclResource] = Encoder.AsObject
+    .instance { r: AclResource =>
+      r.copy(id = iriLens.get(r.id)).void.asJsonObject deepMerge r.value.asJsonObject deepMerge JsonObject(
+        "_path" -> Json.fromString(r.id.string)
+      )
+    }
+    .mapJsonObject(_.remove("_deprecated"))
+
+  implicit val aclWriteResponseEncoder: Encoder.AsObject[ResourceF[AclAddress, Unit]] = Encoder.AsObject
+    .instance { r: ResourceF[AclAddress, Unit] =>
+      r.copy(id = iriLens.get(r.id)).void.asJsonObject deepMerge r.value.asJsonObject
+    }
+    .mapJsonObject(_.remove("_deprecated"))
```

Comment:
We don't need to remove deprecated, we are adding this metadata field to all resources. So basically, you can use the default derived one